app/assets/stylesheets/layout/_navbar.scss

```diff
@@ -0,0 +1,16 @@
+.navbar-top {
+  .navbar-toggle {
+    border-color: $black;
+  }
+  .icon-bar {
+    background-color: $black;
+  }
+  .navbar-brand {
+    height: $navbar-height - $padding-xs-vertical * 2;
+    padding: $padding-xs-vertical $padding-xs-horizontal;
+
+    .brand-image {
```

Comment:
i was adapting it to look like some old version of develop, so my bad. In this case the css also doesn't need to be so bl... complicated